src/main/java/io/github/terra121/projection/GeographicProjection.java

```diff
@@ -277,4 +277,42 @@ public float azimuth(double x, double y, float angle) throws OutOfProjectionBoun
 	public enum Orientation {
 		none, upright, swapped
 	}
+	public enum Facing {
+		N (337.5, 360, "North"),
+		N2 (0, 22.5, "North"),
+		S (157.5, 202.5, "South"),
+		E (67.5, 112.5, "East"),
+		W (247.5, 292.5, "West"),
+		NW (292.5, 337.5, "Northwest"),
+		SW (202.5, 247.5, "Southwest"),
+		SE (112.5, 157.5, "Southeast"),
+		NE (22.5, 67.5, "Northeast"),
+		Unknown (360.5, 10000000.0, "Unknown");
```

Comment:
The java naming convention usually is to name enum instances in uppercase ([Oracle doc](https://docs.oracle.com/javase/tutorial/java/javaOO/enum.html)).
I'll change the `Orientation` enum to reflect that.